Libraries/Components/View/View.js

```diff
@@ -45,19 +45,25 @@ const View: React.AbstractComponent<
       role,
       style,
       tabIndex,
-      ...otherProps
-    }: ViewProps,
-    forwardedRef,
-  ) => {
-    const {
       accessibilityState,
+      accessibilityLabelledBy,
+      accessibilityValue,
       'aria-busy': ariaBusy,
       'aria-checked': ariaChecked,
       'aria-disabled': ariaDisabled,
       'aria-expanded': ariaExpanded,
       'aria-selected': ariaSelected,
-      ...restProps
-    } = otherProps;
+      'aria-labelledby': ariaLabelledBy,
+      'aria-valuemax': ariaValueMax,
+      'aria-valuemin': ariaValueMin,
+      'aria-valuenow': ariaValueNow,
+      'aria-valuetext': ariaValueText,
+      ...otherProps
```

Comment:
Please can we alpha-sort props so it's easier to see the aria props grouped together etc. Same for the props applied to the native component. I'm surprised this isn't a lint rule.